### PR TITLE
core(audit): Add 10ms threshold to bootup-time audits

### LINKED ISSUE #4220: boot-up time shouldn't show values < 10ms
@paulirish mentioned that bootup-time audit had a list of 0ms or really low ms costs. So we want to filter out all requests that have script evaluation and script parse & compile of less than 10ms.

screenshot 2018-01-09 21 17 50

### PR DESCRIPTION
**Pull request by wardpeet**
Fixes #4220

Result:
screenshot 2018-01-09 22 58 23

### CODE REVIEW

File: lighthouse-core/audits/bootup-time.js [hunk 1]
```diff
@@ -73,22 +74,25 @@ class BootupTime extends Audit {
...
 
       // map data in correct format to create a table
-      const results = Array.from(executionTimings).map(([url, groups]) => {
-        // Add up the totalBootupTime for all the taskGroups
-        totalBootupTime += Object.keys(groups).reduce((sum, name) => sum += groups[name], 0);
-        extendedInfo[url] = groups;
+      const results = Array.from(executionTimings)
+        .map(([url, groups]) => {
+          // Add up the totalBootupTime for all the taskGroups
+          totalBootupTime += Object.keys(groups).reduce((sum, name) => sum += groups[name], 0);
```
Comment (patrickhulce) [inline lighthouse-core/audits/bootup-time.js]: @paulirish should we create an "Other" entry in the table so the total still adds up?

Comment (paulirish) [reply lighthouse-core/audits/bootup-time.js]: ehhhh we _could_. wdyt?

Comment (addyosmani) [reply lighthouse-core/audits/bootup-time.js]: I'm in favor of "Other" unless there are better ideas

Comment (paulirish) [reply lighthouse-core/audits/bootup-time.js]: I looked into this and the table already doesn't sum to the total time displayed. the total time shown at the top includes all non-script time (layouts/paints/etc)

so if we really wanted to also capture those, we might do something like 

image

but it's kinda wonky.  

I'm in favor of skipping the "other" row(s)

Comment (patrickhulce) [reply lighthouse-core/audits/bootup-time.js]: Alright that's fine we can skip too

Comment (wardpeet) [reply lighthouse-core/audits/bootup-time.js]: So what's the advice here? 😋

**Pull request merged**